oxide_math/helpers/rectsizehelpers.rs

```diff
@@ -0,0 +1,12 @@
+use crate::components::rect::*;
+use legion::*;
+
+pub fn set_rect_size(world: &mut World, entity: &Entity, width: f32, height: f32) {
+    assert!(width > 0.0f32 && height > 0.0f32);
+    if let Some(mut entry) = world.entry(*entity) {
+        entry.add_component(Rect {
+            width: width,
+            height: height,
```

Comment:
```suggestion
            width,
            height,
```

Reply:
I think the existing code shows that member variables are well initialized than the new code.

Reply:
You may think so, but this is the Rust style.
![image](https://user-images.githubusercontent.com/5622661/161434663-ad96ca2d-9eee-497b-b0c5-06360d6108d2.png)

Reply:
Because format test suggests writing that code in single line, a little modifications applied.

Reply:
Okay. 👍